setup.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python3
 
 from setuptools import setup
+import sys
+
+# Requires python >= 3.3
+assert sys.version_info.major >= 3
+assert sys.version_info.minor >= 3
```

Comment:
I'd much rather have something like:

```python
if sys.version_info < (3,3):
    print("Some message")
    sys.exit(-1)
```

As-is, it still crashes on unsupported pythons. :(